app/src/androidTest/java/com/github/se/signify/ui/screens/UtilsTest.kt

```diff
@@ -244,8 +246,16 @@ class UtilsTest {
     composeTestRule.onNodeWithTag("TopBar").assertIsDisplayed()
   }
 
-  // TODO: test the bottom bar after the refactor of the BottomNavigationMenu()
-  @Test fun bottomBarIsDisplayed() {}
+  @Test
```

Comment:
It is true that the tests do the same, I didn't realize it at first.
The change your are requesting is to remove the one named `bottomBarIsDisplayed()` to avoid duplicate test ?
Tell me if I am correct.

Reply:
Yes absolutely